### PR TITLE
[2.7.0] Add Release notes and update 2.6.0 references

### PR DESCRIPTION
**Pull request by cannischan**
- Adds release notes of 2.7.0
- Update reference from 2.6.0 o 2.7.0

Please review if the release notes and the update to the version numbers make sense

### CODE REVIEW

**Review by mhmxs — APPROVED** [review]
Thanks

**Review by DavidMarchant — APPROVED** [review]

**Comment by hubvu** [comment]
+1 from me!

**Review by hubvu — APPROVED** [review]

File: docs/release-notes.md [hunk 1]
```diff
@@ -7,17 +7,67 @@ weight: 150
...
+- Rewrote the RPC interface between the Control Plane and the Data Plane. All of the old `ctl` tools have been removed and we have added a single `supctl` tool that can be used to debug/program the DP from the command-line
+- Removed the 32-bit mappings and uses the UUIDs passed by the CP directly to address presentations and deployments.
+  > ⚠️ If you decide to upgrade to 2.7.0 and want to downgrade, you can only roll back to 2.6.0, not earlier versions. Roll back instruction can be found [here](/docs/operations/...)
+
+### Fixed
+
+Operator
+
+- Fixed a bug that sometimes caused the operator to enter a deadlock state after Ondat cluster CR object deletion
+- Fixed a bug where the Ondat Operator v2.7.0 does not work with OpenShift 4.7
```
Comment (DavidMarchant) [inline docs/release-notes.md:55]: I don't think this should be here - it's not a change as the issue didn't exist in any released version of the product.

File: docs/release-notes.md [hunk 2]
```diff
@@ -7,17 +7,67 @@ weight: 150
...
+- Adds a feature so when a PVC is not found scheduling will not be blocked
+
+Control Plane
+
+- Set `Only_Numeric_Owners` to true on NFSv4 setting on Ganesha
+
+Data Plane
+
+- Removed support for FUSE. Ondat now only supports TCMU. `target_core_user` must now be used. Read [System Configuration](https://docs.ondat.io/docs/prerequisites/systemconfiguration/) for more information
+- Rewrote the RPC interface between the Control Plane and the Data Plane. All of the old `ctl` tools have been removed and we have added a single `supctl` tool that can be used to debug/program the DP from the command-line
```
Comment (DavidMarchant) [inline docs/release-notes.md:46]: Please don't mention the supctl tool's existence, we explicitly don't want customers touching it without our hands on help.

File: docs/release-notes.md [hunk 3]
```diff
@@ -7,17 +7,67 @@ weight: 150
...
+
+### New
+
+k8s & Orchestrator Rolling Upgrade
+
+- Kubernetes rolling upgrade for AWS EKS, Google Anthos, Google GKE, Microsoft Azure, Openshift and Rancher. Other platforms will be supported in future releases
+
+Operator
+
+- Skipped duplicated nodes by hostname during sync
```
Comment (DavidMarchant) [inline docs/release-notes.md:31]: Just speaking personally, I don't know what this means so I wouldn't expect customers to know either

File: docs/release-notes.md [hunk 4]
```diff
@@ -7,17 +7,67 @@ weight: 150
...
+
+Operator
+
+- Fixed a bug that sometimes caused the operator to enter a deadlock state after Ondat cluster CR object deletion
+- Fixed a bug where the Ondat Operator v2.7.0 does not work with OpenShift 4.7
+
+Control Plane
+
+- Fixed an issue where Ondat was not able to unmount volumes, then occasionally causing volumes to become unhealthy
+- Fixed an issue that caused replicas to go into the “unknown” state during failover
```
Comment (DavidMarchant) [inline docs/release-notes.md:60]: The word 'rarely' should probably feature in this sentence somewhere :smile: I think we're selling ourselves a bit short to new eyes here, likewise with the line above

**Review by DavidMarchant — CHANGES_REQUESTED** [review]
This looks good but I've added a few comments for changes to the release notes

I think more generally this is a departure from the format of the rest of the release notes which have been more general, particularly with bug fixes, & I just want to make sure that it's a conscious departure

**Comment by cannischan** [comment]
@DavidMarchant please review - made changes with your comments

**Review by DavidMarchant — APPROVED** [review]
Thanks @cannischan! lgtm

**Pull request merged**